src/ubuntu/18.04/android/Dockerfile

```diff
@@ -20,3 +21,14 @@ RUN wget https://dl.google.com/android/repository/android-ndk-r21-linux-x86_64.z
     && unzip android-ndk-r21-linux-x86_64.zip -d /usr/local \
     && mv /usr/local/android-ndk-r21 /usr/local/ndk \
     && rm -f android-ndk-r21-linux-x86_64.zip
+
+RUN wget https://dl.google.com/android/repository/commandlinetools-linux-6200805_latest.zip \
+    && mkdir -p /usr/local/android-sdk/cmdline-tools \
+    && unzip commandlinetools-linux-6200805_latest.zip -d /usr/local/android-sdk/cmdline-tools \
+    && rm -f commandlinetools-linux-6200805_latest.zip
+
+# Grab the necessary Android SDK packages / tools
```

Comment:
nit if you need to touch the PR again: move this comment above the download of the commandlinetools to make it clear it's part of the Android SDK, not the NDK.